packages/vega-scenegraph/src/SVGStringRenderer.js

```diff
@@ -330,10 +330,9 @@ function applyStyles(o, mark, tag, defs) {
     if (o.fontWeight) s += 'font-weight: ' + o.fontWeight + '; ';
   }
 
-  for (i=0, n=styleProperties.length; i<n; ++i) {
-    prop = styleProperties[i];
-    name = styles[prop];
-    value = o[prop];
+  for (const prop in styles) {
```

Comment:
Is this going to break compatibility with older browsers?

Reply:
It looks like it's well supported https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Browser_compatibility.